packages/forma-36-react-components/src/components/Dropdown/DropdownList/DropdownList.tsx

```diff
@@ -8,7 +8,7 @@ export type DropdownListProps = {
   className?: string;
   testId?: string;
   border?: 'top' | 'bottom';
-  maxHeight?: number;
+  maxHeight?: 'initial' | 'inherit' | string | number;
```

Comment:
@marcolink would `string | number` not be enough? I am wondering about adding `'initial' | 'inherit' `...

Reply:
@burakukula 
technically it would, but this gives you a better autocomplete hint support.

Reply:
@marcolink I'm not convinced, but if that make life easier then sure thing. 🙂

Reply:
@marcolink @burakukula we have [csstype](https://www.npmjs.com/package/csstype), I believe we can do `number |  CSS.MaxHeightProperty<string>` similar to usage in the [grid](https://github.com/contentful/forma-36/blob/master/packages/forma-36-react-components/src/components/Grid/Grid.tsx#L2) component

Reply:
@mshaaban0 I changed it accordingly to your suggestion 👍